static/index.html

```diff
@@ -136,25 +136,25 @@
     }
     function external (url) {
       if (url.match(/\.(gif|jpg|jpeg|png)$/)) {
-        return stash("<img src=\"" + url.replace(/^http:\/wiki\//,'http://c2.com/wiki/') + "\">")
```

Comment:
If the `url` passed in here can be prefixed with `http` or `https` then this replace might not work properly.